examples/basic/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0"
+
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.63.0"
```

Comment:
the resource group module requires >=1.64.0, so you should probably change this back to use latest provider version, or update all the modules in this repo to use `>=1.64.0`

Reply:
updated all the modules in this repo to use >=1.64.0